rsmtool/utils.py

```diff
@@ -784,9 +785,11 @@ def quadratic_weighted_kappa(y_true_observed, y_pred, ddof=0):
     y_pred_var, y_pred_avg = (np.var(y_pred, ddof=ddof),
                               np.mean(y_pred))
 
-    numerator = np.mean((y_true_observed - y_pred)**2)
+    y_true_array = np.array(y_true_observed)
+    y_pred_array = np.array(y_pred)
+    numerator = 2 * np.cov(y_true_array, y_pred_array, bias=True)[0][1]
```

Comment:
Since we allow user to override the default ddof, we want to use `ddof=ddof` here rather than `bias=True` (the result will be the same when `ddof=0`)